server/pulp/server/webservices/views/root_actions.py

```diff
@@ -0,0 +1,27 @@
+from django.views.generic import View
+
+from pulp.server.auth import authorization
+from pulp.server.managers import factory
+from pulp.server.webservices.controllers.decorators import auth_required
+from pulp.server.webservices.views.util import generate_json_response
+
+
+class LoginView(View):
+    """
+    View for API root-level actions.
+    """
+
+    @auth_required(authorization.READ)
+    def post(self, request, *args, **kwargs):
+        """
+        Return client SSL certificate and a private key
+
+        :param request: WSGI request object
+        :type request : WSGIRequest
+        :return       : Response containing cert and key
```

Comment:
Consider not vertically aligning these colons. It visually looks nice but then when you rename one variable you may have to adjust the indentation on multiple lines.